### PR TITLE
Nerfs Sheltered (Understand Common)

### PR DESCRIPTION
**Pull request by JamieD1**
# Github documenting my PR
Nerfs Sheltered - Can now Understand Common but still cant speak it.
# Wiki Documentation
🆑
Tweak: Nerfs Sheltered - Can now Understand Common but still cant speak it.
/🆑

### CODE REVIEW

**Comment by patpol4** [comment]
what's the point now

**Review by Bibby0110 — DISMISSED** [review]
Template

**Comment by maxion12345** [comment]
tf you mean nerf it's a buff

**Comment by patpol4** [comment]
Nerf to being able to be an ass and powergame not speaking

File: code/datums/traits/negative.dm [hunk 1]
```diff
@@ -606,7 +606,7 @@
 
 /datum/quirk/sheltered/on_spawn()
 	var/mob/living/carbon/human/H = quirk_holder
-	H.remove_language(/datum/language/common)
+	H.remove_language(/datum/language/common, FALSE, TRUE)
```
Comment (Bibby0110) [inline code/datums/traits/negative.dm]:
```suggestion
	H.remove_language(/datum/language/common, TRUE, FALSE)
```

**Review by Bibby0110 — APPROVED** [review]

**Review by Bibby0110 — APPROVED** [review]

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**